charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -0,0 +1,156 @@
+  # minAvailable: 1
+  # maxUnavailable: 1
+  labels: {}
+
+extraArgs:
+  loggerFormat: json
+
+# -- Additional environment variables (ex.: secret tokens, flags)
+env: []
+
+# -- Additional hostPath mounts
+extraHostPathMounts: []
+  # - name: certs-dir
+  #   mountPath: /etc/kubernetes/certs
+  #   subPath: ""
+  #   hostPath: /etc/kubernetes/certs
+#   readOnly: true
+
+# -- Extra Volumes for the pod
+extraVolumes: []
+  # - name: example
+  #   configMap:
+#     name: example
+
+# -- Extra Volume Mounts for the container
+extraVolumeMounts: []
+  # - name: example
+#   mountPath: /example
+
+extraContainers: []
+  # - name: config-reloader
+#   image: reloader-image
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+  runAsGroup: 1000
+
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+#   memory: 128Mi
+
+# -- Annotations to be added to the deployment
+annotations: {}
+
+# -- Annotations to be added to pod
+podAnnotations: {}
+
+# -- NodeSelector configurations. Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# -- Tolerations configurations. Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+tolerations: []
+  # - key: "key"
+  #   operator: "Equal|Exists"
+#   value: "value"
+#   effect: "NoSchedule|PreferNoSchedule"
+
+# -- Affinity configurations
+affinity: {}
+
+serviceMonitor:
+  enabled: false
+  extraLabels: {}
+  annotations: {}
+
+# -- should be true and means that you have the legal right to run a vmgateway
+# that can either be a signed contract or an email with confirmation to run the service in a trial period
+# https://victoriametrics.com/legal/eula/
+eula: false
+
+remote:
+  read:
+    url: "http://single-victoria-metrics-single-server.default.svc.cluster.local:8428"
+    tenant: "" # Format: vm_account_id:vm_project_id (for example: 0:0)
+    basicAuth:
+      username: ""
+      password: ""
+  write:
+    url: "http://single-victoria-metrics-single-server.default.svc.cluster.local:8428"
+    tenant: "" # Format: vm_account_id:vm_project_id (for example: 0:0)
+    basicAuth:
+      username: ""
+      password: ""
+    metric_names:
+      anomaly_score: anomaly_score
+      yhat: yhat
+    extra_labels:
+      model: zscore
+
+scheduler:
+  class: "scheduler.periodic.PeriodicScheduler"
+  fit_every: "1d"
+  fit_window: "7d"
+  infer_every: "10s"
+
+model:
+  enabled: "zscore" # Available options: zscore, prophet, holt_winters
```

Comment:
user can enable more than one model in that case we must create multiple containers